pylab/website/tests/test_comments.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+from django_webtest import WebTest
+
+from django.contrib.auth.models import User
+
+from django_comments.models import Comment
+
+from pylab.core.models import Project
+
+
+class CommentsTests(WebTest):
+
+    def setUp(self):
+        self.user = User.objects.create_user('u1')
+        self.project = Project.objects.create(
+            author=self.user,
+            title='Test project',
+            description='Description',
+            created='2015-08-13'
+        )
+
+    def test_add_comment(self):
+        resp = self.app.get('/projects/test-project/', user=self.user)
+        now = datetime.datetime.now()
+        comment = Comment.objects.create(
```

Comment:
All forms work in same way. I found two small problems, why the comment is not saved. 
1. I don't know why, but ```django_comments``` requires that user, who is commenting should have valid email (I found out this by debugging). So when creating user you should add email parameter, e.g. ```User.objects.create_user('u1', email='test@example.com')```. 
2. ```self.app.get()``` method's ```user``` parameter is username, not User instance. I agree, that this naming is confusing. You could create ```self.username``` variable and pass it, instead of ```self.user```.